### PR TITLE
FEATURE: BEND-33 Save new photo endpoint

### PR DESCRIPTION
**Pull request by octapf**
- Create Save new photo endpoint
- add route
- add controller
- add action
- add error & success msg
- add endpoint

### CODE REVIEW

File: src/http/controller/User.ts [hunk 1]
```diff
@@ -611,4 +611,61 @@ export module UserController {
...
+   * @param res {express.Response}
+   * @param next {express.NextFunction}
+   */
+  export const CreatePhoto = async (
+    req: express.Request,
+    res: express.Response,
+    next: express.NextFunction
+  ) => {
+    const userId = JWTMiddleware._getRightUserId(req);
+    const photo: PhotoId = req.body;
```
Comment (octapf) [inline src/http/controller/User.ts:628]: @whitehatdevv is it correct the way I'm declaring this const (photo: PhotoId) ? As i saw in another case you use a DTOManager but i don't get how entirely works.

File: src/http/actions/User.ts [hunk 2]
```diff
@@ -497,4 +497,38 @@ export module UserActions {
...
+   * @param photo {PhotoId}
+   * @param f {Function} (err?: [ErrHandlerBody], scss?: [SuccessHandlerBody])
+   */
+  export function CreatePhotoAction(
+    userId: string,
+    photo: PhotoId,
+    f: Function
+  ) {
+    //TODO Connect to S3
+    User.findOneAndUpdate({_id: userId}, {$push: {profilePhoto: photo}})
```
Comment (octapf) [inline src/http/actions/User.ts:514]: @whitehatdevv I found this way to push into the profilePhoto array the new doc (photo) but i was unsure if it is the proper one.

**Review by PatyVilas — APPROVED** [review]

File: src/http/actions/User.ts (hunk 2 above)
Comment (whitehatdevv) [reply src/http/actions/User.ts:514]: Yes this is the right one.

File: src/http/controller/User.ts (hunk 1 above)
Comment (whitehatdevv) [reply src/http/controller/User.ts:628]: Yes

**Review by whitehatdevv — DISMISSED** [review]

**Comment by whitehatdevv** [comment]
@PatyVilas @octapf please ix the conflicts.

**Review by PatyVilas — APPROVED** [review]

**Pull request merged**